### DISCUSSION TITLE
Chrome apps have similar icon

### DISCUSSION THREAD

**Discussion by kostiato** [SylEleuth/gruvbox-plus-icon-pack#123] [Q&A]
First, let me thank you for the greatest and the most complete icon theme I've seen.
I have a very strange behavior in KDE Plasma 6.2.4 (Wayland):
- all Chrome apps in application launcher/dashboard have a similar look
- default Chrome icon is shown for started app too 
- window rules are also ignored

Could you please help me to find a workaround?

![image](https://github.com/user-attachments/assets/062b6e07-1875-4c52-bc3b-df31c12bf9d2)
---
![image](https://github.com/user-attachments/assets/4fd97977-7ff9-4434-b5b3-ba12502d5e02)

**Comment by SylEleuth** [comment]
What icons they should have? You blurred the names so I have no idea what I'm looking at.

PS. Considering `apps` icons it's not most complete. Still hundreds if not over thousand icons are missing.

**Reply by kostiato** [reply]
It doesn't actually depend on app name. 
Each application has default chrome icon: Teams, Slack, Skype, YouTube, Telegram...

**Comment by SylEleuth** [comment]
I'm not familiar with chrome apps. What are they, how do You install them?

**Reply by kostiato** [reply]
In Chrome menu > Cast, save, and share > Install page as app...
After that in .desktop file Icon is defined as _chrome-appId-ProfileName_.
I've managed to resolve the issue by changing all .desktop Icon values from _chrome-appId-ProfileName_ icons to native icons. Looks good now.
By the way, while searching for icon all chrome apps looks good in a selection dialog but after applying they look like chrome icon.
I was just wondering why Breeze didn't override chrome apps icons...

**Comment by SylEleuth** [comment]
I've installed Chrome and tried it with Youtube and I've got an app with a proper icon. You still see the chrome icon, am I right?

I've installed Twitter and icon is missing. It's easy to fix, just create symlink to an icon. But I need a name of it. So if you give me names of icons that You are missing I will add them .

**Reply by kostiato** [reply]
Thank you for investigation.
Now, I see proper icons for all apps because I updated all .desktop files in `~/.local/share/applications`.  That, basically, was the quick workaround a was looking for.
I think that, when you have multiple profiles in Chrome just symlinking won't work because every newly installed app under a different profile has that profile in its name. That's why YouTube also worked for me with Default profile, but didn't work with others than Default.

**Reply by SylEleuth** [reply]
I'm not sure about profiles in chrome and how it affects names of the apps. But chrome apps icons are in the pack, they have their own name. When You turn the page into an app You will have an option e.g. "Install Youtube". It's not working that way with every page, so I believe it's limited to specific applications. And if some icons are missing they should be added. I just need their names. At least for test.

**Reply by kostiato** [reply]
Teams, Skype, Slack, Telegram...
How can I identify chrome apps icons in the pack?

**Reply by SylEleuth** [reply]
It's in `.desktop` file, `icon=`record.
